app/__init__.py

```diff
@@ -124,7 +118,6 @@ def override_url_for():  # pylint: disable=unused-variable
     login_manager.init_app(application)
 
     if settings.EQ_DEV_MODE:
-        # TODO fix health check so it no longer sends message to queue
```

Comment:
https://trello.com/c/5WpLqNYy/364-health-check-rabbitmq-without-putting-a-message-on-the-queue